lib/controllers/coinstac/consortia.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const config = require('config'); // jshint ignore:line
+const _ = require('lodash');
+const consortiaDbOptions = config.get('coinstac.pouchdb.consortia');
+
+const internals = {};
+
+internals.consortiaSchema = joi.object().keys({
+    label: joi.string().required(),
+    users: joi.array().required(),
+    description: joi.string().required(),
+    tags: joi.array().required(),
+    analyses: joi.array().required()
+});
+
+internals.savedConsortiaSchema = internals.consortiaSchema.keys({
```

Comment:
I wasn't sure, so I tried it, and it appears to work. If it didn't append, then the tests would fail because output validation would detect keys in the response object that are not in the joi schema. 

Looking at the docs, the answer is more obvious: https://github.com/hapijs/joi#objectkeysschema
>Sets or extends the allowed object keys where:

>schema - optional object where each key is assigned a joi type object. If schema is {} no keys allowed. If schema is null or undefined, any key allowed. If schema is an object with keys, the keys are added to any previously defined keys (but narrows the selection if all keys previously allowed). Defaults to 'undefined' which allows any child key.
>```
>var base = Joi.object().keys({
>    a: Joi.number(),
>    b: Joi.string()
>});
>// Validate keys a, b and c.
>var extended = base.keys({
>    c: Joi.boolean()
>});
>```